### PR TITLE
Updated EHRbase Multistage Dockerfile to use postgres:13.3-alpine

### PR DESCRIPTION
**Pull request by wlad**
## Changes

- Updated Dockerfile to use postgres:13.3-alpine
- removed old db setup scripts

## Related issue

closes https://github.com/ehrbase/project_management/issues/545

## Additional information and checks

- [ ] Pull request linked in changelog

### CODE REVIEW

File: base/db-setup/cloud-db-setup.sql [hunk 1]
```diff
@@ -36,54 +38,59 @@ DECLARE
   holder2 record;
   pg_version integer;
 BEGIN
-  -- version 0.2.0
+  -- version 0.4.0
```
Comment (jakesmolka) [inline base/db-setup/cloud-db-setup.sql:41]: Cool, so this worked? Great!

File: tests/Taskfile.yml [hunk 2]
```diff
@@ -143,17 +181,19 @@ tasks:
...
-      - robot -e future -e circleci -e TODO -e obsolete -e libtest -d results/xml -L TRACE --noncritical not-ready --log None --report None --output ehrsta.xml --name EHRSTATUS robot/EHR_STATUS_TESTS
-      - robot -e future -e circleci -e TODO -e obsolete -e libtest -d results/xml -L TRACE --noncritical not-ready --log None --report None --output knowld.xml --name KNOWLEDGE robot/KNOWLEDGE_TESTS
-      - robot -e future -e circleci -e TODO -e obsolete -e libtest -d results/xml -L TRACE --noncritical not-ready --log None --report None --output aqledb.xml --name "QUERY empty_db"  -i empty_db  robot/QUERY_SERVICE_TESTS
-      - robot -e future -e circleci -e TODO -e obsolete -e libtest -d results/xml -L TRACE --noncritical not-ready --log None --report None --output aqlldb.xml --name "QUERY loaded_db" -i loaded_db robot/QUERY_SERVICE_TESTS
+      - robot -e future -e circleci -e TODO -e obsolete -e libtest -d results/xml -L TRACE --skiponfailure not-ready --log None --report None --output compop.xml --name COMPO robot/COMPOSITION_TESTS
```
Comment (jakesmolka) [inline tests/Taskfile.yml:184]: Most changes on this file are looking like format changes. But changes down here are looking semantic. Just a note, that I really can't review them as I don't know the meaning of the flags.

File: Dockerfile [hunk 3]
```diff
@@ -1,89 +1,40 @@
...
-
-EXPOSE 5432
+RUN mkdir -p ${PGDATA}; \
+    chown postgres: ${PGDATA}; \
+    chmod 0700 ${PGDATA}; \
+    su - postgres -c "initdb -D ${PGDATA}"
+
+# COPY DB SETUP SCRIPT
+# START DB AND LET THE SCRIPT DO ALL REQUIRED CONFIGURATION
+COPY base/db-setup/cloud-db-setup.sql /postgres/cloud-db-setup.sql
```
Comment (jakesmolka) [inline Dockerfile:20]: Couldn't this use `COPY scripts/db-setup.sql /docker-entrypoint-initdb.d/` too?

**Review by jakesmolka — COMMENTED** [review]
Looks good, but also see my inline comments.

And I was wondering if you have created the issue regarding testing the Docker Hub image of EHRbase.

**Comment by wlad** [comment]
> And I was wondering if you have created the issue regarding testing the Docker Hub image of EHRbase.

created it today, see #542

**Pull request merged**